lib/companion_web/jobs/process_opt_out.ex

```diff
@@ -42,13 +42,14 @@ defmodule Companion.Jobs.ProcessOptOut do
     msisdn <> "^^^ZAF^TEL"
   end
 
-  defp openhim_optout_from_contact(contact) do
+  defp openhim_optout_from_contact(contact, id) do
     %{
       "urns" => [nurse_urn | _],
       "fields" => %{
         "opt_out_date" => opt_out_date,
         "registered_by" => device_msisdn,
-        "facility_code" => facility_code
+        "facility_code" => facility_code,
+        "contact_id" => contact_id
```

Comment:
I don't think that `contact_id` is a field on the RapidPro contact. I think the `uuid` field is probably what you're looking for here. The RapidPro response looks something like:
```
"uuid": "a49fddb7-cde0-4d3a-aa24-33ecc826f0d2"
"urns": ["tel:+27821234567"],
"fields": {
          registered_by: "+27820000000",
          facility_code: "123456",
          opt_out_date: "2018-05-30T15:10:32.650440Z"
          opt_out_date: "2018-05-30T15:10:32.650440Z",
        }
```

Reply:
oh I see, okay will make the change